### PR TITLE
Resource Graph: Use ObjectArray format option (includes SDK update)

### PR DESCRIPTION
**Pull request by OlhaTkachenko**
Resource-graph extension consumes objectArray response format. For this it was required to use latest resource-graph sdk.
Tested locally.
Version will be updated in index, when more changes added

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/resource-graph/azext_resourcegraph/vendored_sdks/resourcegraph/version.py [hunk 1]
```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.6.0"
+VERSION = "2.0.0"
```
Comment (chiragg4u) [inline src/resource-graph/azext_resourcegraph/vendored_sdks/resourcegraph/version.py]: We should release 1.0.0 version for PS/CLI as there was no major version before. Also, we are not doing any change which breaks backward compat.

**Comment by chiragg4u** [comment]
Suggest differentiating the files which are changed by you vs. the one which was generated. It will help in review.

File: src/resource-graph/azext_resourcegraph/vendored_sdks/resourcegraph/version.py (hunk 1 above)
Comment (OlhaTkachenko) [reply src/resource-graph/azext_resourcegraph/vendored_sdks/resourcegraph/version.py]: It is python sdk

**Comment by OlhaTkachenko** [comment]
> Suggest differentiating the files which are changed by you vs. the one which was generated. It will help in review.

Commit "SDK update" includes all not my changes, two other are my changes. Please let me know if it isn't good enough

**Comment by chiragg4u** [comment]
Changes look good. Thanks, Olha.

**Pull request merged**